modules/yarn/src/main/resources/ignite.properties

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
Do we need this file? It's copied from `core`.

Reply:
Yes, we need this file since ignite-yarn does not depend on ignire-core: It's just a deployment mechanism.

Reply:
ok, got it.